app/controllers/monsters_controller.rb

```diff
@@ -0,0 +1,6 @@
+class MonstersController < ApplicationController
+
+	def index
```

Comment:
It also appears we don't have any ERB templates using this controller, so this can be removed

Reply:
done